responsys-api.gemspec

```diff
@@ -17,15 +17,15 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 1.9.3'
+  spec.required_ruby_version = '>= 2.1.0'
```

Comment:
Not ready for this yet. It's a breaking change that will prevent 1.9 apps to use future versions of the GEM. I intend to do that leap as part of the next REST rewrite. Please revert this change.